be/src/storage/rowset_update_state.cpp

```diff
@@ -45,24 +45,19 @@ Status RowsetUpdateState::load(Tablet* tablet, Rowset* rowset) {
     return _status;
 }
 
-Status RowsetUpdateState::_do_load(Tablet* tablet, Rowset* rowset) {
-    auto span = Tracer::Instance().start_trace_txn_tablet("rowset_update_state_load", rowset->txn_id(),
-                                                          tablet->tablet_id());
-    _tablet_id = tablet->tablet_id();
-    auto& schema = rowset->schema();
-    vector<uint32_t> pk_columns;
-    for (size_t i = 0; i < schema.num_key_columns(); i++) {
-        pk_columns.push_back((uint32_t)i);
-    }
-    vectorized::Schema pkey_schema = ChunkHelper::convert_schema_to_format_v2(schema, pk_columns);
-    std::unique_ptr<vectorized::Column> pk_column;
-    if (!PrimaryKeyEncoder::create_column(pkey_schema, &pk_column).ok()) {
-        CHECK(false) << "create column for primary key encoder failed";
+Status RowsetUpdateState::_load_deletes(Rowset* rowset, uint32_t delete_id, vectorized::Column* pk_column) {
```

Comment:
The 100M data is small to test the OOM. You can follower the test in jira to test.
I only test 800M, OOM throws. The total data size is 280 billion(42TB) to ingest in 10 nodes.

Reply:
> The 100M data is small to test the OOM. You can follower the test in jira to test. I only test 800M, OOM throws. The total data size is 280 billion(42TB) to ingest in 10 nodes.

If data size is very large, OOM still throws. What I want to express is this pr can reduce memory usage but it can't resolve 
all OOM problems because oom has many reasons to cause. I think we can do this test after another problems are resloved.

Reply:
The memory consumption of primary-key table is a bottle point. I think we should have tests of higher priority and test in advance.